LTTngDataExtensions/Tables/ExecutionEvent.cs

```diff
@@ -176,10 +176,8 @@ public static void BuildTable(ITableBuilder tableBuilder, IDataExtensionRetrieva
                 InitialFilterQuery = filterIdleSamplesQuery,
             };
 
-            timelineByCPUTableConfig.AddColumnRole(ColumnRole.EndThreadId, nextTidColumn);
```

Comment:
The EndThreadId and WaitEndTime column roles are plugin specific and were removed. Were these in this plugin on purpose?

Reply:
The Unit Tests would not cover these changes. Did you try to test this LTTng table and see if anything was affected? I assume they were on purpose if in the code but may not break anything super specific. Please check CPU % graph display. If these are plugin specific, what is the equivalent to do this in a plugin specific manner? I don't think there is, and this usage was already plugin specific - so maybe using the wording "plugin specific" is not good wording?

Also, does this mean xperf ETW will stop configuring/using EndThreadId since I believe it does so for Windows CPU Scheduling events?

Reply:
The column roles were changed to strings, so if these are needed the roles can be replaced with "EndThreadId" and "WaitEndTime"

Loading the tables in WPA doesn't show any problem with these changes.